roles/ceph-rgw-loadbalancer/templates/haproxy.cfg.j2

```diff
@@ -43,7 +43,7 @@ frontend rgw-frontend
 backend rgw-backend
     option forwardfor
     balance static-rr
-    option httpchk Get /
+    option httpchk GET /
 {% for host in groups[rgw_group_name] %}
 {% for instance in hostvars[host]['rgw_instances'] %}
 	server {{ 'server-' + hostvars[host]['ansible_hostname'] + '-' + instance['instance_name'] }} {{ instance['radosgw_address'] }}:{{ instance['radosgw_frontend_port'] }} weight 100
```

Comment:
You should still be adding `check` at the end of this line in order for HAproxy to actually do the HTTP GET requests.